### PR TITLE
Added support for the <br> tag and support for nested assets in the t…

### PR DESCRIPTION
**Pull request by sunil-lakshman**

### CODE REVIEW

File: lib/contentstack_utils/model/options.rb [hunk 1]
```diff
@@ -85,6 +87,8 @@ def render_node(node_type, node, inner_html)
                     renderString = "<li>#{inner_html}</li>"
                 when 'hr'
                     renderString = "<hr />"
+                when 'br'
```
Comment (ishaileshmishra) [inline lib/contentstack_utils/model/options.rb]: Remove this br from the code.

File: lib/contentstack_utils/model/options.rb [hunk 2]
```diff
@@ -48,6 +48,8 @@ def render_mark(mark_type, text)
                     renderString = "<sub>#{text}</sub>"
                 when 'superscript'
                     renderString = "<sup>#{text}</sup>"
+                when 'br'
```
Comment (ishaileshmishra) [inline lib/contentstack_utils/model/options.rb]: Code implementation is missing

**Comment by ishaileshmishra** [comment]
- Test cases missing, Please add test cases for the break line. 
- Code missing for image
- break code implementation missing

**Review by ishaileshmishra — COMMENTED** [review]
Please refer the comment

**Comment by sunil-lakshman** [comment]
> * Test cases missing, Please add test cases for the break line.
> * Code missing for image
> * break code implementation missing

Code implementation done and added test cases.

File: lib/contentstack_utils/model/options.rb (hunk 2 above)
Comment (sunil-lakshman) [reply lib/contentstack_utils/model/options.rb]: Done

File: spec/mock/json_to_html_mock.rb [hunk 3]
```diff
@@ -43,6 +43,7 @@
 EntryReferenceLinkJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"7626ea98e0e95d602210","type":"reference","attrs":{"target":"_self","href":"/copy-of-entry-final-02","display-type":"link","entry-uid":"entry_uid_2","content-type-uid":"embeddedrte","locale":"en-us","type":"entry","class-name":"embedded-entry"},"children":[{"text":"/copy-of-entry-final-02"}]}],"type":"doc"}'
 EntryReferenceInlineJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"506 4878f3f46 s21f0cbc aff","type":"reference","attrs":{"display-type":"inline","entry-uid":"entry_uid_3","content-type-uid":"embeddedrte","locale":"en-us","type":"entry","class-name":"embedded-entry"},"children":[{"text":""}]}],"type":"doc"}'
 HRJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"f5a7b57 40a8a5c3 576828276b","type":"hr","children":[{"text":""}],"attrs":{ }}],"type":"doc"}'
+BRJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"f5a7b57 40a8a5c3 576828276b","type":"br","children":[{"text":""}],"attrs":{ }}],"type":"doc"}'
```
Comment (ishaileshmishra) [inline spec/mock/json_to_html_mock.rb]: the br tag comes in the text not as type. similar to other **mark types** like **subscripts** and **bold**

**Review by ishaileshmishra — COMMENTED** [review]
Added on minor change

File: spec/mock/json_to_html_mock.rb (hunk 3 above)
Comment (sunil-lakshman) [reply spec/mock/json_to_html_mock.rb]: Done

**Comment by ishaileshmishra** [comment]
Reviewed, found code implementation missing. Please complete the code implementation @sunil-lakshman

**Comment by sunil-lakshman** [comment]
> Reviewed, found code implementation missing. Please complete the code implementation @sunil-lakshman

Done

File: lib/contentstack_utils/version.rb [hunk 4]
```diff
@@ -1,3 +1,3 @@
 module ContentstackUtils
-    VERSION = "1.1.3"
+    VERSION = "1.1.4"
```
Comment (ishaileshmishra) [inline lib/contentstack_utils/version.rb]: Make it **1.2.0**. Since we are adding one more function support for the MarkType.

**Review by ishaileshmishra — APPROVED** [review]
Approved.

Please share the pull request (PR) details with Gautam for the CAB and also submit a request to the SRE team for code review.

**Review by ishaileshmishra — APPROVED** [review]
approved

**Pull request merged**